### DISCUSSION TITLE
Does cobbler supports provisioning in a shared-lom setup?

### DISCUSSION THREAD

**Discussion by Shubhangi-dell** [cobbler/cobbler#3029] [Q&A] [answered]
Hi! I am trying to provision OS in a shared lom setup but it is throwing errors as
![image](https://user-images.githubusercontent.com/72869337/163391506-18379de9-8d07-457b-be0d-3f1187fd0066.png)
I have made the --server-override changes as well for my profile as mentioned in the docs.
So, does cobbler supports provisioning in a shared-lom setup, or Am I missing something?

**Comment by SchoolGuy** [comment, marked as answer]
Yes it should be supported. If not then this is something we definitely want to implement. In theory if I don't get it wrong, this should just be a second interface inside a system and you should be good to go.

**Reply by Shubhangi-dell** [reply]
No, its like a single nic with multiple IPs as shown in the below pic: 
![image](https://user-images.githubusercontent.com/72869337/163424361-01e6599a-cbe6-45b1-aafb-909e5a2412b1.png)
So, will cobbler work on this?

**Reply by SchoolGuy** [reply]
I haven't had the time to try it out myself. However it is try that at the moment Cobbler only contains the logic of one IP address per interface (which is especially stupid in case of IPv6). I know I have machines at work from Dell where I can try this out. Gimme some days please so I can confirm if it is really possible or not.

**Reply by Shubhangi-dell** [reply]
I have actually tested this and its working fine. The only thing we have to do is modify the dhcp.template accordingly and it will run smoothly. Maybe you can add this as well as one of your feature :)
